Dockerfile

```
@@ -8,6 +8,7 @@ RUN apk add --no-cache -t build-dependencies git make gcc g++ python libtool aut
     && npm install -g node-gyp
 
 COPY ./package.json ./package-lock.json ./init-account-lookup.sql ./init-central-ledger.sql /opt/als-oracle-pathfinder/
+COPY ./src /opt/als-oracle-pathfinder/src
```

The advantage of this is that lower layers in the image change less frequently, which is better for storage (pretty unimportant) and build speed (can be important for development turn-around)
```suggestion
COPY ./src/lib/pathfinder/package.json ./src/lib/pathfinder/package-lock.json /opt/als-oracle-pathfinder/src/lib/pathfinder/
```

---

If you accept this, you'll need to revert the deletion of this line later in the file.

---

I would prefer to stick to what is currently there as that is the way "other" mojaloop images are built using this pattern.